_extensions/sdca-theme/_extension.yml

```diff
@@ -21,6 +21,18 @@ contributes:
       search:
         location: navbar
         type: overlay
+      navbar:
```

Comment:
I think there might be issues with this. Could you test it out by also using a navbar item in the root `_quarto.yml`? So that we can confirm it doesn't overwrite it?